drivers/java/driver.go

```diff
@@ -318,8 +318,9 @@ func (d *Driver) StartTask(cfg *drivers.TaskConfig) (*drivers.TaskHandle, *drive
 
 	pluginLogFile := filepath.Join(cfg.TaskDir().Dir, "executor.out")
 	executorConfig := &executor.ExecutorConfig{
-		LogFile:  pluginLogFile,
-		LogLevel: "debug",
+		LogFile:     pluginLogFile,
+		LogLevel:    "debug",
+		FSIsolation: capabilities.FSIsolation == drivers.FSIsolationChroot,
```

Comment:
Huh, when did this field become a bool? Seems like we should rename it to something chroot specific if all it's just a boolean flag for chrooting or not.

Not a blocker for this PR as it's unrelated, but it's the first I'm noticing this API.

Reply:
In the executor side, this has been a boolean for a really long time, as long back as when it was introduced [1], and it survived when we switched to new executor[2]

[1] https://github.com/hashicorp/nomad/commit/f67a237361#diff-2ac060857b3fa786648cf40f184a228bR25
[2] https://github.com/hashicorp/nomad/commit/5b14d24bf4626bab420d00783d92bcf25e0b641e#diff-bd8eafff490626bcacaa44c9c0c3de13R168

Reply:
Ha! Nice spelunking. Kind of strange we named an enum and bool flag the same thing, but that's not really relevant here!